Dockerfile

```diff
@@ -4,8 +4,12 @@ RUN apk add --update nodejs npm
 
 WORKDIR /app
 
-COPY . .
+COPY package.json package-lock.json ./
+COPY scripts ./scripts
+COPY abis ./abis
```

Comment:
Why this if we are using dockerignore and you below you do copy the whole directory?

Reply:
By doing this, the step `npm install` can be cached on a layer and won't be rebuilt as long as the files required by the installation process are not modified (package.json, package-lock.json, scripts, and abis).

In other words, you can modify the source code and the step `npm install` won't be rebuilt.

Reply:
This was needed yep.